### ISSUE TITLE
New version of fable-library

### ISSUE THREAD

**Issue by nojaf** [fable-compiler/Fable#3614]
### Description

I noticed there was a new release of `Fable.Core` on NuGet, but there didn't seem to be any release of [fable-library](https://www.npmjs.com/package/fable-library) on npm.

**Comment by MangelMaxime** [comment]
This is "normal", currently the only person that can push packages to `fable-library` is @alfonsogarciacaro.

I opened a ticket to NPM to request transfer of the ownership of `@fable-compiler` org and `fable-library` package. I opened the ticket on the Oct 18 but still don't have new on that subject.

They said that such request can take a long time because they need to be careful about ownership transfer.

I tried to contact Alfonso by email didn't get an answer either.

**Comment by nojaf** [comment]
Yeah, I was afraid of this response.
Thanks for explaining the current situation.

**Comment by laurentpayot** [comment]
@nojaf beware of https://github.com/elmish/hmr/issues/44

**Comment by MangelMaxime** [comment]
@laurentpayot Fable.Core and `fable-library` are not related. I believe no one is actually using `fable-library` from NPM packages in their project as this is a really niche feature that was not really made public yet. Will be made public when I have access to the `fable-library` NPM package.

Also, I am using Elmish.HMR with latest version of Fable without problem so I need to have a look at how you are using it in your case and what is different.

**Comment by nojaf** [comment]
> I believe no one is actually using fable-library from NPM packages in their project

🖐️ Oh, I do! But I get what you are saying. What is the latest status of this? If nothing seems to be happening could we consider using a different name maybe?

**Comment by ncave** [comment]
`fable-library-js` seems unused.

**Comment by MangelMaxime** [comment]
> 🖐️ Oh, I do! But I get what you are saying.

We found the ONE 😝 (just kidding).

I am still waiting an answer from NPM/Github support but I don't think it will happen.

We could go with `fable-library-js` or we could create a new org to host everything under it. Problems is that `@fable-compiler` is taken already by Alfonso.

We could go with `@fable_compiler` or `@fable-compiler-org` or `@real-fable-compiler`. 

I personally would prefer to go with a organisation but like that I can make people maintainers for the whole organisation and avoid forgetting to add someone to a package.

**Comment by nojaf** [comment]
Do you need the fancy `@` thing to publish as an organisation?

**Comment by MangelMaxime** [comment]
@nojaf Yes this is how NPM identify an organisation / scopes so yes this is how it is going to be.

To give you an exemple of how package will be named: https://www.npmjs.com/search?q=@mangelmaxime

**Comment by nojaf** [comment]
I see, well I really have a strong preference on the naming here.
`@fable-community` has a nice ring but really whatever works for you.

**Comment by MangelMaxime** [comment]
`@fable-community` is a good one too and different enough from `@fable-compiler` which we don't have access to ATM.

To me, it just convey that is more like something made by the "community" than official Fable itself but that's not a big deal.

If I was to have 2 propositions it would be `@fable-community` or `@fable-org`?

Any of these two would do for me and would unstuck us from the current situation at least. Meaning that we could release `fable-library` and also move all of Fable packages to that org.

@dbrattli @ncave What do you think?

**Comment by MangelMaxime** [comment]
Packages are under `@fable-org`
